Code/Editor/IEditor.h

```diff
@@ -168,8 +168,6 @@ enum EEditorNotifyEvent
     eNotify_OnVegetationObjectSelection, // When vegetation objects selection change.
     eNotify_OnVegetationPanelUpdate,   // When vegetation objects selection change.
 
-    eNotify_OnDisplayRenderUpdate,     // Sent when editor finish terrain texture generation.
-
```

Comment:
This is legacy viewport/editor code that is no longer used - there's so much more to be removed but at least these incremental changes move us in the right direction